libs/payments/cart/src/gql/tax-address.model.ts

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Field, ObjectType } from '@nestjs/graphql';
+
+@ObjectType({
+  description: 'The Tax Address associated with the Cart',
+})
+export class TaxAddress {
+  @Field({ nullable: true, description: 'Error reason ID.' })
```

Comment:
s/Error reason ID/Country code for tax/

Reply:
Uhm...whoops 😅 Thanks for spotting and the text!